pkg/inject/server.go

```diff
@@ -88,28 +87,35 @@ func (whsvr *WebhookServer) mutate(ar *v1beta1.AdmissionReview) *v1beta1.Admissi
 		}
 	}
 
-    injectType, err := getAnnotation(&pod.ObjectMeta, annotationInjectTypeKey)
+	injectType, err := getAnnotation(&pod.ObjectMeta, annotationInjectTypeKey)
     containerMode, err := getAnnotation(&pod.ObjectMeta, annotationContainerModeKey)
```

Comment:
this and the next line have some funky indentation

Reply:
Good spot, fixed.